### ISSUE TITLE
Arduino Shield Interest

### ISSUE THREAD

**Issue by mtongnz** [mtongnz/mtDueDMX#1]
If you're interested in the Arduino Due shield I'm designing, leave a comment here.  It should be available towards the end of this year.

**Comment by NeforMalchik** [comment]
yes

**Comment by ghost** [comment]
I'm interested :D

**Comment by AndColla** [comment]
I'm interested

**Comment by ghost** [comment]
When do you thing the DMX Shield is going to be available?

**Comment by thepersonamusic** [comment]
I too am interested!

**Comment by mtongnz** [comment]
Work has been crazy the last couple of months and will be for the next couple.  I'm hoping to have the finalized design done in the next month.  Prototype production and testing will probably take about a month.  All going well, the first boards should be available Feb/March.

I'm planning to offer the board in a few variants - either with electrical isolation or without.  Both will be available with or without the network connection also for those who don't need it.

Sorry for the wait.  It'll be worth it!

**Comment by mtongnz** [comment]
So I've got the new PCBs.  Just waiting on 1 slow part (the optocouplers) to arrive.  Once they get here, I'll be assembling 10 test boards, giving them a quick test to ensure things are working correctly before selecting a few testers.

**Final pricing** will be as follows (USD):
 $89 - Electrically and optically isolated board, 4 bi-directional ports, 10/100 RJ45 ethernet port, unique MAC address chip
$69 - Same as above but without ethernet or MAC chip
$69 - Optically isolated board, 4 bi-directional ports, 10/100 RJ45 ethernet port, unique MAC address chip, no electrical isolation
$59 - Same as above but without ethernet or MAC chip

**Testers**
I'm thinking I'll sell the first small run on a first come basis here.  They'll get the full board ($89 value) at only $69 (limit 2 each).  Testers will hopefully be competent with Arduino (preferably Due experience) and be able to check the mtDueDMX library prior to me releasing the finished boards for general sale.

Let me know your thoughts on pricing.  I think it compares very well as single port isolated DMX boards are $45 each and ethernet shields without MAC chip are around $15.

**Comment by amcanderson** [comment]
I too am interested!

**Comment by alterloewe** [comment]
Me too.....

**Comment by philarnaud** [comment]
Thank you for your very good work. I am interesting in this shield for Arduino Due. Tell me when it's available.
I am working with a DMX grove shield (very small card) wich is working well on Uno or Mega with DmwSimple library. Unfortunaltely it was not working on DUE and this is normal I have tried with your library, I made exactly what you said (changes in vector.cpp) and it was compiling well.  But when I connect a PAR led with Power in 4 and Red in 5, and I put these 2 values to 255, the small led on the PAR led is well blinking which is indicating that a signal is read, but Red is not showing up on the PAR Led. Do you have any idea of what complementary tests  I can do to diagnose the issue?
Best regards
Philippe Arnaud

**Comment by philarnaud** [comment]
Any news?

**Comment by mtongnz** [comment]
Sorry for the delays... Work is very busy, as well as a few other personal commitments I've had.
I've got the first board built a and have been doing testing for the past couple of weeks, streamlining my testing methods for multiple boards at once.  I'm planning to build the other boards this weekend, aiming to have them on sale March 6th.  This will be a very limited run and aimed mainly for testing - hence they'll have lower prices.
The next run will be a much larger order and will be selling at the prices above.  It'll likely be on sale mid April as I'm heading away for a month in March.

**Comment by mtongnz** [comment]
@philarnaud regarding your issue above. Please open a new issue and provide your code. Cheers

**Comment by Ventustium** [comment]
Me too, I'm very interested. Thank You

**Comment by wasn-eu** [comment]
would like to get one test shield.
i am programming microcontrollers for years now.

**Comment by mtongnz** [comment]
Apologies. Due to a family matter, I ran out of time to get these up for sale before heading off on my holiday. So unfortunately, it'll be another 4 weeks. Really sorry about this. I did get them built and tested. They seem to be working great and I'll be looking at various pcba options while I'm away

**Comment by ghost** [comment]
Hey, when do you think the boards are going to be ready?

**Comment by marin-8** [comment]
Any updates? :(

(I was very interested in this project...)

**Comment by dmxlights** [comment]
Any update on these boards. I am also intrested in some
